### DISCUSSION TITLE
Plotman is showing 1:6, 1:5 in phases - where there is only 4 phases

### DISCUSSION THREAD

**Discussion by windkostya** [ericaltendorf/plotman#588] [Q&A] [answered]
Can someone please explain me is there anything wrong with my plotman config or it just not working properly. 

There 4 stages in chia plotting. For some reason when my plotman is running is see 1:5, 3:6 and i don;t understand how this is possible since there only 4 stages. 

Please help.

Thank you,
Cons
![chia-plotting](https://user-images.githubusercontent.com/17420563/118868186-bec42f00-b898-11eb-86fb-de5589fce444.png)

**Comment by windkostya** [comment]
This have been resolved. Thank you for sharing this link: https://github.com/ericaltendorf/plotman/issues/151#issuecomment-832980275.

**Comment by mruckli** [comment, marked as answer]
Plotting consists of 4 phases but each phase can have sub-phases. The first number stands for the phase and the second for the sub-phase.
In your case 3:6 equals to phase 3, sub-phase 6 (almost done).
More detailed info to be found here: https://chiadecentral.com/plotman-automate-and-manage-chia-plotting/

**Reply by windkostya** [reply]
thank you so much for helping to understand it more detailed. Fantastic.

**Reply by lnb2g** [reply]
phrase:step
